test/helpers/index.ts

```diff
@@ -46,3 +47,13 @@ export const resetDatabase = async (mongoose) => {
       await db.dropCollection(collectionName)
     })
 }
+
+export const generateTokenHelper = ({ secret }) => {
+  const generateTokenResult = generateToken(secret)
+  expect(generateTokenResult).toBeTruthy()
```

Comment:
> please don't test helpers (except),

Okay, I've removed the expect.

> just use `return generateTokenResult.token` also else is not necessary

Actually I have to use `generateTokenResult?.token` otherwise there's a TS warning due to the object possibly being null. That's the reason why I created the helper, so that the null checking would be done here and won't be needed in multiple places.

Reply:
if you wan to call the fail is ok (although is not necessary) but else is not necessary:

```js
export const generateTokenHelper = ({ secret }) => {
  const generateTokenResult = generateToken(secret)

  if (generateTokenResult) {
    return generateTokenResult.token
  }

  fail("generateToken returned null")
}
```
or just 

```js
export const generateTokenHelper = ({ secret }) => {
  const { token } = generateToken(secret)
  return token
}
```

Reply:
Yes you're right, I somehow missed that I had a return in the if block.

I've refactored it now.

However your second suggestion will not work due to the same reason I mentioned above, and I get this error -> `Property 'token' does not exist on type '{ token: string; } | null'.ts(2339)`
The reason being generateTokenResult returns either an object with a token field, or null.